cmd/utils/device_id.go

```diff
@@ -139,3 +161,31 @@ func computeDeviceIDFromBytesF(command *cobra.Command, args []string) error {
 	fmt.Println(deviceID)
 	return nil
 }
+
+func toUUIDDeviceIDF(command *cobra.Command, args []string) error {
+	deviceID := args[0]
+	if !utils.IsValidAstarteDeviceID(deviceID) {
+		fmt.Println("%s is not a valid Astarte Deivce ID\n", deviceID)
+		os.Exit(1)
+	}
+
+	deviceUUID, err := utils.DeviceIDToUUID(deviceID)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println(deviceUUID)
+	return nil
+}
+
+func fromUUIDDeviceIDF(command *cobra.Command, args []string) error {
```

Comment:
Maybe check if the arg is a valid UUID? I'd do a uuid.Parse here, which is cheap enough, which fails early if the UUID isn't valid.